configure.ac

```diff
@@ -21,6 +21,9 @@ AC_CANONICAL_HOST
 : ${CFLAGS="-O3 -Wall -Werror -g"}
 
 # Checks for programs.
+AC_CHECK_PROG(ABIDIFF, abidiff, abidiff, no)
+AS_IF([test "x$ABIDIFF" = xno],
+      [AC_MSG_ERROR([Please install abidiff (tool from libabigail).])])
```

Comment:
Are there any issues preventing the usage of `AX_WITH_PROG(ABIDIFF, abidiff)`?

Reply:
I just didn't see the need for the extra `--with-$TOOL` option that comes with it. But I can add that to the PR.

Reply:
Also, just using `AX_WITH_PROG(ABIDIFF, abidiff)` doesn't make the build fail at configure time if `abidiff` is not available.

Reply:
Ack.  Got it.
I'm resolving this conversation.